blog/2023-07-20-reproducibility-designer.md

```diff
@@ -0,0 +1,43 @@
+---
+author: Antonin Delpeuch
+title: "We're hiring a designer!"
+---
+
+OpenRefine is hiring a designer to help shape our improvements to OpenRefine's operation history and test those changes with user panels.
+<!--truncate-->
+
+[OpenRefine](https://openrefine.org/) is a power tool for working with messy data, popular in a diverse range of communities. It has been serving the needs of journalists, librarians, Wikipedians, scientists for more than 13 years and is taught in many curricula and workshops around the world. OpenRefine is a fiscally sponsored project of [Code for Science & Society Inc](https://codeforscience.org/), a 501(c)(3) charitable organization in the USA.
+
+OpenRefine has long had a rudimentary system JSON scripts which can be used to re-execute series of operations on a project. As part of [a project funded by the Essential Open Source Software for Science programme](https://chanzuckerberg.com/eoss/proposals/improving-openrefines-reproducibility/), we are working on replacing this by a proper notion of OpenRefine *workflows* (or *macros*, *recipes*…), with a better usability and appropriate reproducibility guarantees.
```

Comment:
I took a crack at drafting an alternative for the opening which reflects how I think of the Undo history, as well as removing the JSON acronym to avoid scaring off those who are intimidated by it.
```suggestion
OpenRefine has long had an Undo History & Redo capability which users have repurposed to execute operation histories, potentially modified, on the same or different projects. As part of [a project funded by the Essential Open Source Software for Science program](https://chanzuckerberg.com/eoss/proposals/improving-openrefines-reproducibility/), we are working on replacing this by a proper notion of OpenRefine *workflows* (or *macros*, *recipes*…), with a better usability and appropriate reproducibility guarantees.
```

Reply:
That looks good to me. In addition to this change, perhaps it'd be worth moving the entire paragraph further down, as something that readers could optionally have a look at to have a deeper understanding of the project's scope. Would that make sense @lozanaross?

Reply:
Yes, this was my original thinking. Keep the technical details, because they are useful to give a sense of the work, but move it a bit further down, so that in general we start with the bullet points of what we're looking for and what are the requirements, and mention like "for details see below", etc.